docs/UpdaterConfig.md

```diff
@@ -223,17 +223,27 @@ The information is downloaded in a single HTTP request and polled regularly.
 <!-- vehicle-positions BEGIN -->
 <!-- NOTE! This section is auto-generated. Do not change, change doc in code instead. -->
 
-| Config Parameter           |       Type      | Summary                                                                    |  Req./Opt. | Default Value | Since |
-|----------------------------|:---------------:|----------------------------------------------------------------------------|:----------:|---------------|:-----:|
-| type = "vehicle-positions" |      `enum`     | The type of the updater.                                                   | *Required* |               |  1.5  |
-| feedId                     |     `string`    | Feed ID to which the update should be applied.                             | *Required* |               |  2.2  |
-| frequency                  |    `duration`   | How often the positions should be updated.                                 | *Optional* | `"PT1M"`      |  2.2  |
-| url                        |      `uri`      | The URL of GTFS-RT protobuf HTTP resource to download the positions from.  | *Required* |               |  2.2  |
-| [headers](#u__6__headers)  | `map of string` | HTTP headers to add to the request. Any header key, value can be inserted. | *Optional* |               |  2.3  |
+| Config Parameter            |       Type      | Summary                                                                    |  Req./Opt. | Default Value | Since |
+|-----------------------------|:---------------:|----------------------------------------------------------------------------|:----------:|---------------|:-----:|
+| type = "vehicle-positions"  |      `enum`     | The type of the updater.                                                   | *Required* |               |  1.5  |
+| feedId                      |     `string`    | Feed ID to which the update should be applied.                             | *Required* |               |  2.2  |
+| frequency                   |    `duration`   | How often the positions should be updated.                                 | *Optional* | `"PT1M"`      |  2.2  |
```

Comment:
Btw is 1 minute default for a vehicle position updater reasonable?

Reply:
In the US it is. :) 

There they are happy with a 5 minute update frequency from their source.

Reply:
I would be ok with changing the default.

Reply:
I think we can keep the current value.